.binstar.yml

```diff
@@ -0,0 +1,47 @@
+package: dask
+user: blaze
+
+platform: 
+  - linux-64
+  - linux-32
+  - osx-64
+  - win-32
+  - win-64
+
+engine:
+  - python=2.6
+  - python=2.7
+  - python=3.3
+  - python=3.4
+
+before_script:
+  - python -V
+
+install:
+  - conda config --add channels blaze
+
+script:
+  - conda build conda.recipe
+
+env:
+  - LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
+
+iotimeout: 120
+
+build_targets: conda
+
+notifications:
+  email:
+    recipients: ['mrocklin@continuum.io', 'phillip.cloud@continuum.io',
+                 'bgriffith@continuum.io']
+
+---
+platform: win-32
+engine: python=2.6
+env: LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
+exclude: true
```

Comment:
This was to deal with a default encoding that wasn't specified on binstar on osx and linux64 IIRC. I'd leave it off for now and add it only if necessary.